modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/manager/v1_0/test/DefaultObjectEntryManagerImplTest.java

```diff
@@ -2175,6 +2175,16 @@ public void testGetObjectEntries() throws Exception {
 			).build(),
 			childObjectEntry1);
 
+		String pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'";
+
+		testGetObjectEntries(
```

Comment:
I think we are testing this already in the `buildRangeExpression`. Could you check?

Reply:
yes, true, I just applied the new one to apply specifically the reproducing steps 😅 Also, the other test does not check if the filtering works correctly, as only 2 elements are created and both appear in the result of the request. Just saying, if not needed I can remove it as a redundant case.

Reply:
Could we re-define the existing test by just passing the `childObjectEntry2.getDateCreated()` and checking just the `childObjectEntry2` is being returned?

Reply:
With that, we would be reusing the existing code and also we would be validating the filter works.

Reply:
Anyway, AFAIK, the existing test should fail too for PostgreSQL database, because the filter is also being used in the existing tests. Maybe it is a matter of executing the tests for PostgreSQL database. Could you confirm?

Reply:
ok, will confirm and remodel accordingly!